app/views/journeys/nics/Class4ExemptionCategoryView.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.common.{BusinessId, TaxYear, UserType}
+@import cats.implicits._
+@import controllers.journeys.nics.routes.Class4ExemptionCategoryController
+@import views.html.templates.helpers.Heading
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Content
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.checkboxes.{CheckboxItem, Checkboxes}
+
+@this(layout: templates.Layout,
+      formHelper: FormWithCSRF,
+      govukErrorSummary: GovukErrorSummary,
+      govukCheckboxes: GovukCheckboxes,
+      govukButton: GovukButton,
+      heading: Heading)
+
+@(form: Form[_], taxYear: TaxYear, businessId: BusinessId, userType: UserType, mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages(s"class4ExemptionCategory.title.$userType"))) {
+
+    @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+    }
+
+    @heading(messages(s"class4ExemptionCategory.title.$userType"), None, extraClasses = "no-padding-top govuk-!-margin-bottom-3")
+
+    @formHelper(action = Class4ExemptionCategoryController.onSubmit(taxYear, businessId, mode), 'autoComplete -> "off") {
+
+        @govukButton(ButtonViewModel(messages("site.continue")))
+
```

Comment:
There are some templates that are a bit cleaner we can use. I'd have a look at the profit and loss journey to see how the views and page objects are set up there

Reply:
I wasn't aware we had all those clean components to choose from. I followed the design of a page with similar structure but looks like that was outdated.
<img width="1081" alt="Screenshot 2024-07-08 at 21 44 54" src="https://github.com/hmrc/income-tax-self-employment-frontend/assets/89045034/3363d55b-f7f7-4ad4-881d-ff7410bb0cc9">

Reply:
We haven't had a chance yet to go back and make a lot of the older pages consistent with the new ones. It's a fairly minor point but worth changing if you're making changes anyway.
I'll probably make a few quick PRs this sprint to add some of these consistencies.